docs/schemas/weblate-userdata.schema.json

```diff
@@ -80,7 +80,7 @@
         },
         "suggested": {
           "$id": "#root/profile/suggested",
-          "title": "Number of sugested strings",
+          "title": "Number of suggested strings",
```

Comment:
This one is maintained externally, I've made the change there: https://github.com/WeblateOrg/weblate_schemas/commit/29e05e7e418ea031f269e3cc6fd55849fc46d890